core/zoe/docs/zoe.md

```diff
@@ -0,0 +1,263 @@
+# Zoe: Offer-Safety Enforcement
+
+Note: Zoe is currently at the pre-alpha stage. It has not yet been
+formally tested or hardened.
+
+This guide assumes some knowledge of the [ERTP fundamentals](./README.md).
+
+## What is Zoe? 
+
+__For users__: Zoe guarantees that as a user of a smart contract, you
+will either get what you wanted or get a refund back, even if the
+smart contract is buggy or backdoored. (In fact, the smart contract
+never has access to your digital assets.)
+
+__For developers__: Zoe provides a safety net so you can focus on what
+your smart contract does best, without worrying about your users
```

Comment:
I would say "you can focus on what is central to your contract". "Focus on what you do best" makes sense when we know that the listener has an established position that they want to leverage.

Reply:
hmm, I think you're right that it's presumptive. I think it's snappier than the alternative, though